### ISSUE TITLE
[BUG] Error: INSTALLATION FAILED: chart requires kubeVersion: < 1.25.0-0 which is incompatible with Kubernetes v1.25.5

### ISSUE THREAD

**Issue by kamzata** [rancher/rancher#40389]
**Rancher Server Setup**
- Rancher version: -
- Installation option: Helm v3.11.0

**Information about the Cluster**
- Kubernetes version:
    - Client Version: v1.25.4
    - Kustomize Version: v4.5.7
    - Server Version: v1.25.5
- Cluster Type (Local/Downstream): Local, VMs on Proxmox

**Describe the bug**

```
kamzata@k8s-master:~$ h install rancher rancher-latest/rancher -n cattle-system --set hostname=rancher.example.com
Error: INSTALLATION FAILED: chart requires kubeVersion: < 1.25.0-0 which is incompatible with Kubernetes v1.25.5
```

**Expected Result**

Rancher installed

**Additional context**

I'm using a Cloud-Init version of Ubuntu 22.04 in some Proxmox virtual machines which now delivers Kubernetes 1.26.1 version. How could I use Rancher in a production environment with those version discrepancies? I mean... should I use another cloud-init distro or something for safely run Rancher?

**Comment by bpedersen2** [comment]
Yes you need to run on a k8s cluster <1.25 (check cloud-init docs for how to select the right k8s install).  Ideally this cluster should only run rancher, the workloads got to the managed clusters.

As a side note, unless you are developing rancher you should install from rancher-stable,  rancher-latest does not yet contain important fixes and installs just a release candidate.

**Comment by kamzata** [comment]
> Yes you need to run on a k8s cluster <1.25 (check cloud-init docs for how to select the right k8s install). Ideally this cluster should only run rancher, the workloads got to the managed clusters.
> 
> As a side note, unless you are developing rancher you should install from rancher-stable, rancher-latest does not yet contain important fixes and installs just a release candidate.

Thanks. Can Rancher handle clusters >1.25?

**Comment by David-VTUK** [comment]
> > Yes you need to run on a k8s cluster <1.25 (check cloud-init docs for how to select the right k8s install). Ideally this cluster should only run rancher, the workloads got to the managed clusters.
> > As a side note, unless you are developing rancher you should install from rancher-stable, rancher-latest does not yet contain important fixes and installs just a release candidate.
> 
> Thanks. Can Rancher handle clusters >1.25?

Currently, no. Officially supported versions are listed in our support matrix : https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-1/

A future release of Rancher will include support for 1.25. This is currently being tracked in https://github.com/rancher/rancher/issues/38701

**Comment by omni52** [comment]
I mention my solution here https://github.com/rancher/rancher/issues/43092#issuecomment-2423135070
